### PR TITLE
Fix `yarn create` command on Windows

### PR DESCRIPTION
**Pull request by Volune**
**Summary**

Yarn create fails on Windows, see #3530 
The issue is caused by Yarn trying to run a javascript file as an executable (the !# doesn't work in Windows command)

This PR uses the existing `linkBin` function to create any appropriate shim depending on the system.

**Test plan**

No existing test. Manually tested on Windows 10 and one linux. More tests by other people would be appreciated.
Also any idea to automate a test here is welcomed.

### CODE REVIEW

File: src/cli/commands/create.js [hunk 1]
```diff
@@ -51,7 +52,10 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       throw new MessageError(reporter.lang('createInvalidBin', packageName));
     }
 
-    await child.spawn(binPath, rest, {stdio: `inherit`});
+    const linkedBinPath = config.getTemp(`create-bin-${packageName}`);
+    await linkBin(binPath, linkedBinPath);
```
Comment (arcanis) [inline src/cli/commands/create.js]: I'm not sure calling `linkBin` is required, is it? It should have been done when adding the package (l30)

Comment (Volune) [inline src/cli/commands/create.js]: I assumed we were previously not using the already linked bin for a reason. If there are no reasons, I can indeed change to use the existing one in global.

Comment (arcanis) [inline src/cli/commands/create.js]: Nope, no particular reason - I just forgot about this behavior when I implemented the feature 😞

Comment (Volune) [inline src/cli/commands/create.js]: @arcanis Please review the new implementation. Tested on Windows.
Also I wonder, shouldn't we pass the flags to `runGlobal` (and now also to `getBinFolder`)?

**Pull request merged**